docs/usage/config-validation.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
We have a "Troubleshooting Renovate" page here: https://docs.renovatebot.com/troubleshooting/

We should insert a link to your new "config validation page" here. 😄